docs/releases/2.0.txt

```diff
@@ -250,6 +254,14 @@ Database backend API
   ``datetime_extract_sql()`` methods now return only the SQL to perform the
   operation instead of SQL and a list of parameters.
 
+* Increased the number of rows fetched by :meth:`.QuerySet.iterator` at each
+  iteration from 100 to 2000. The old behavior can be restored using the
+  ``chunk_size`` parameter. For instance, considering a ``Book`` model:
```

Comment:
There are actually several lexers for Django in pygments. The one I used was http://pygments.org/docs/lexers/#pygments.lexers.templates.DjangoLexer and that was incorrect. You can see the other lexers defined for Django on page.

I'll submit a revised version of the patch soon that uses `::`.